### PR TITLE
feat: Provide access to features via Event Dashboard

### LINKED ISSUE #1761: Provide access to mentioned features via Event Dashboard instead of navigation drawer
**Describe the feature you'd like**
Addition of a box above the section of 'Gross Sales' in Event Dashboard with the following items:
- Check In
- Orders
- Tickets
- Edit Event
- Event Settings

Orders, Tickets and Edit Event should show only in Developer Mode.

Parent issue: #1745 

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1761 

**Changes:** 

Added a box above the section of 'Gross Sales' in Event Dashboard, containing the following items:

- Check In
- Orders
- Tickets
- Edit Event
- Event Settings

Removed these items from the navigation drawer.

Orders, Tickets and Edit Event would show only in Developer Mode.

**Screenshot for the change:**

Provide access to features via Event Dashboard

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Change the color from blue please. And implement as standard list items ensuring that clicks ripple etc

**Comment by ShridharGoel** [comment]
@iamareebjamal Should I implement a RecyclerView or just wrap each layout in a CardView?

**Comment by iamareebjamal** [comment]
No need of any of that. There is just an attribute to make it ripple

**Comment by ShridharGoel** [comment]
@iamareebjamal Updated.

File: app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java [hunk 1]
```diff
@@ -97,6 +105,39 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 openChart.putExtra(EVENT_ID, initialEventId);
                 startActivity(openChart);
             });
+
+        SharedPreferences sharedPreferences = getActivity().getSharedPreferences(Constants.FOSS_PREFS, MODE_PRIVATE);
+
+        if (sharedPreferences.getBoolean(getString(R.string.developer_mode_key), false)) {
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java]: Move the logic from view to presenter/VM

File: app/src/main/java/com/eventyay/organizer/core/main/FragmentNavigator.java [hunk 2]
```diff
@@ -72,18 +68,6 @@ void loadFragment(int navItemId) {
...
-                break;
-            case R.id.nav_tickets:
-                fragment = TicketsFragment.newInstance(eventId);
-                break;
-            case R.id.nav_orders:
-                fragment = OrdersFragment.newInstance(eventId);
-                break;
-            case R.id.nav_event_settings:
-                fragment = EventSettingsFragment.newInstance(eventId);
-                break;
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/main/FragmentNavigator.java]: Don't remove

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/main/FragmentNavigator.java]: We have to keep them in the navigation drawer also?

**Comment by iamareebjamal** [comment]
Don't remove, nowhere in the parent issue is it mentioned to remove the navigation menu items

File: app/src/main/java/com/eventyay/organizer/core/main/FragmentNavigator.java (hunk 2 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/main/FragmentNavigator.java]: Updated.

File: app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java]: Done.

**Pull request merged**